tests/data/wp_die.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SzepeViktor\PHPStan\WordPress\Tests;
+
+use function PHPStan\Testing\assertType;
+
+/** @var array $array */
+$array = null;
+
+$true = ['exit' => true];
+$false = ['exit' => false];
+
+assertType('*NEVER*', wp_die('', ''));
+assertType('*NEVER*', wp_die('', '', $true));
+assertType('void', wp_die('', '', $false));
```

Comment:
Could we simplify this?

```suggestion
assertType('*NEVER*', wp_die('', ''));
assertType('*NEVER*', wp_die('', '', ['exit' => true]));
assertType('void', wp_die('', '', ['exit' => false]));
```